admin/client/src/components/FormAction/FormAction.js

```diff
@@ -113,10 +161,4 @@ FormAction.propTypes = {
   extraClass: React.PropTypes.string,
 };
 
-FormAction.defaultProps = {
```

Comment:
- type is now passed in through attributes nested
- bootstrapButtonStyle doesn't have a set default anymore; It depends on the name to determine fallback (save action will have a different style to other actions).
- disabled; I'll put this back though. ;P